src/commands/update-github-release.ts

```diff
@@ -217,11 +217,11 @@ async function createNewRelease(
     prerelease: isPrerelease
   });
 
-  const success = response.status === 200;
+  const success = response.status === 201;
```

Comment:
Wäre es hier nicht klüger auf beide Status Codes zu horchen, oder kann ein 200er hier garnicht auftreten?

```suggestion
  const success = response.status === 200 || response.status === 201;
```

Reply:
Also das release wurde nur erfolgreich erstellt wenn der Status 201 ist. 
Es ist bestimmt möglich dass die Response einen anderen Status enthält, aber dann war es halt nicht erfolgreich

Reply:
Ich frage halt, da hier vorher explizit auf "200" geprüft wurde. Aber ich denke mal, das war dann einfach bisher falsch?

Reply:
Ja genau 👍